src/main/eo/org/eolang/math/number.eo

```diff
@@ -151,6 +153,27 @@
         value
         value.neg
 
+  # Round ^.n up
+  [] > ceil
+    ^.n > value!
+    if. > @
+      or.
+        is-nan
+        is-int
+        eq.
+          value
+          positive-infinity
+        eq.
+          value
+          negative-infinity
+      value
+      []
```

Comment:
@maxonfjvipon Why do we need this abstract object here? To improve readability?

Reply:
@Graur I'm not sure. Rather for optimization purposes (caching `as-int`). If you don't like it I can do it without an abstract object

Reply:
@maxonfjvipon Just asking. I think we can keep it as is